__tests__/app.test.js

```diff
@@ -80,12 +80,36 @@ describe('GET /api/articles', () => {
             expect(response.body.message).toBe("No Article Found under Article ID 99")
         })
     })
-    test("GET status 400 and error message when an invalid id is given", () => {
+    test("GET status 400 and error message when an invalid article_id is given", () => {
         return request(app)
         .get("/api/articles/not-an-id")
         .expect(400)
         .then((response) => {
             expect(response.body.message).toBe("Bad Request")
         })
     })
+});
+describe('GET /api/articles', () => {
+    test("GET /api/articles returns status 200 and returns an array of article data to the client", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            const body = response.body
+            expect(body.length).toBe(13);
+            body.forEach((article) => {
+                expect(article).toEqual({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    created_at: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(String)
+                })
+            })
+        })
+    })
```

Comment:
You need to assert the articles are in order of date, I recommend jestSorted for this